wdl/src/main/scala/wdl/WdlWorkflow.scala

```diff
@@ -155,15 +155,21 @@ case class WdlWorkflow(unqualifiedName: String,
     leftOutputs.find(_.unqualifiedName == name)
   }
 
-  lazy val hasEmptyOutputSection = workflowOutputWildcards.isEmpty && children.collect({ case o: WorkflowOutput => o }).isEmpty
+  lazy val noWorkflowOutputs = children.collect { case o: WorkflowOutput => o }.isEmpty
+  lazy val noOutputWildcards = workflowOutputWildcards.isEmpty
+
+  lazy val hasEmptyOutputSection =  noWorkflowOutputs && noOutputWildcards
+
+  lazy val isTopLevelWorkflow: Boolean =
```

Comment:
In general I've been trying to be good about not chiming in on such quibbles but can we not have this on two lines at least? :)

Reply:
Oh yeah, this was to make it easier to target the debugger, should be reverted